token/js/src/instructions/associatedTokenAccount.ts

```diff
@@ -88,3 +88,44 @@ function buildAssociatedTokenAccountInstruction(
         data: instructionData,
     });
 }
+
+/**
+ * Construct a RecoverNested instruction
+ *
+ * @param nestedAssociatedToken             Nested associated token account (must be owned by `ownerAssociatedToken`)
+ * @param nestedMint                        Token mint for the nested associated token account
+ * @param destinationAssociatedToken        Wallet's associated token account
+ * @param ownerAssociatedToken              Owner associated token account address (must be owned by `owner`)
+ * @param ownerMint                         Token mint for the owner associated token account
+ * @param owner                             Wallet address for the owner associated token account
+ * @param programId                         SPL Token program account
+ * @param associatedTokenProgramId          SPL Associated Token program account
+ *
+ * @return Instruction to add to a transaction
+ */
+export function recoverNestedInstruction(
```

Comment:
nit for consistency with others
```suggestion
export function createRecoverNestedInstruction(
```

Reply:
this one fixed here [92d370f](https://github.com/solana-labs/solana-program-library/pull/4508/commits/92d370f656849cacf0180caee89ed68087ab31f5)